packages/react-reconciler/src/ReactFiberHostContext.js

```diff
@@ -79,7 +82,12 @@ function pushHostContext(fiber: Fiber): void {
     rootInstanceStackCursor.current,
   );
   const context: HostContext = requiredContext(contextStackCursor.current);
-  const nextContext = getChildHostContext(context, fiber.type, rootInstance);
+  let type = fiber.type;
+  // For React event components, we get the typeof field (the symbol)
```

Comment:
Why?

Reply:
Because we then don't need to expose the fiber work tag logic in the renderer for the fiber.